src/components/editor/markdown-renderer/replace-components/quote-options/quote-options.tsx

```diff
@@ -18,26 +17,26 @@ const findQuoteOptionsParent = (nodes: DomElement[]): DomElement | undefined =>
   })
 }
 
-const getElementReplacement = (node: DomElement, index: number, counterMap: Map<string, number>, nodeConverter: SubNodeConverter): (ReactElement | undefined) => {
-  if (node.name !== 'blockquote' || !node.children || node.children.length < 1) {
-    return
-  }
-  const paragraph = findQuoteOptionsParent(node.children)
-  if (!paragraph) {
-    return
-  }
-  const childElements = paragraph.children || []
-  const optionsTag = childElements.find(isColorExtraElement)
-  if (!optionsTag) {
-    return
-  }
-  paragraph.children = childElements.filter(elem => !isColorExtraElement(elem))
-  const attributes = optionsTag.attribs
-  if (!attributes || !attributes['data-color']) {
-    return
+export class QuoteOptionsReplacer implements ComponentReplacer {
```

Comment:
this file should then be called `quote-options-replacer.tsx`?

Reply:
renamed